engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -640,6 +640,19 @@ protected NetworkOrchestrator() {
         setStateMachine();
     }
 
+    private void updateNetworkDetails(NetworkVO networkPersisted, Network network) {
```

Comment:
may be _updateRouterIpInNetworkDetails(networkId, routerIp, routerIpv6)_ , not required to pass entire network object.